src/command/account/login.ts

```diff
@@ -0,0 +1,79 @@
+import { Argument, LeafCommand, Option } from 'furious-commander'
+import { createSpinner } from '../../utils/spinner'
+import { VerbosityLevel } from '../root-command/command-log'
+import {
+  AccountCommand,
+  MAX_PASSWORD_LENGTH,
+  MAX_USERNAME_LENGTH,
+  MIN_PASSWORD_LENGTH,
+  MIN_USERNAME_LENGTH,
+} from './account-command'
+import { Message } from '../../utils/message'
+import { CommandLineError } from '../../utils/error'
+import { getFieldOrNull } from '../../utils'
+import { assertBytes } from '../../utils/types'
+
+export class Login extends AccountCommand implements LeafCommand {
+  public readonly name = 'login'
+
+  public readonly description = 'Login to a portable FDP account'
+
+  @Argument({
+    key: 'username',
+    description: 'Username for the portable FDP account',
+    type: 'string',
+    required: true,
+    minimumLength: MIN_USERNAME_LENGTH,
+    maximumLength: MAX_USERNAME_LENGTH,
+  })
+  public username!: string
+
+  @Option({
+    key: 'portable-password',
+    description: 'Password for the portable FDP account',
+    type: 'string',
+    required: { when: 'quiet' },
+    minimumLength: MIN_PASSWORD_LENGTH,
+    maximumLength: MAX_PASSWORD_LENGTH,
+  })
+  public portablePassword!: string
+
+  public async run(): Promise<void> {
+    await super.init()
+    this.validateBeeDebugAvailable()
```

Comment:
why is it necessary? read operations are required only with bee for login.

Reply:
This is needed to initialize fdp-storage. With the new version, this is no longer necessary.